docs/overview.md

```diff
@@ -11,18 +11,18 @@ Essentially, it is an alternative to running the `renovate` CLI tool, with the f
 - Webhook listener to enable dynamic reactions to repository events
 - Administration APIs for probing the system state or triggering jobs
 
-## Mend Renovate features
+## Mend Renovate Self-hosted features
```

Comment:
Risks confusion with CLI

Reply:
Updated all references to "Mend Renovate Self-hosted App(s)"